pkgs/stdenv/generic/default.nix

```diff
@@ -223,7 +223,7 @@ let
         # include it in the result, it *is* available to nix-env for queries.
         meta = { }
             # If the packager hasn't specified `outputsToInstall`, choose a default,
-            # namely `p.bin or p.out or p`;
+            # namely `lib.getBin p`;
```

Comment:
Hm, indeed, but `getBin` would try `pkg.bin`, `pkg.out` and then just `pkg`. There, however, we'll try `pkg.bin`, `pkg.out`, `pkg.{foo,bar}` (where `outs = [ "foo" "bar" ]`) and then return `null` (without trying `pkg`).